README.md

```diff
@@ -205,31 +119,20 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **mrprotocoll**
```

Comment:
- I suggest that you remove your partner in the ReadMe for this special branch because this activity is individual.